storage/engine/mvcc.go

```diff
@@ -251,41 +209,51 @@ func updateStatsOnResolve(ms *proto.MVCCStats, key proto.Key, origMetaKeySize, o
 func updateStatsOnAbort(ms *proto.MVCCStats, key proto.Key, origMetaKeySize, origMetaValSize,
 	restoredMetaKeySize, restoredMetaValSize int64, orig, restored *proto.MVCCMetadata,
 	origAgeSeconds, restoredAgeSeconds int64) {
-	if !updateStatsForKey(ms, key) {
+	ok, sys := updateStatsForKey(ms, key)
+	if !ok {
 		return
 	}
 	origTotalBytes := orig.KeyBytes + orig.ValBytes + origMetaKeySize + origMetaValSize
-	if !orig.Deleted {
-		ms.LiveBytes -= origTotalBytes
-		ms.LiveCount--
+	if sys {
+		ms.SysBytes -= origTotalBytes
```

Comment:
No, for the system keys we're not differentiating between live and dead bytes. It's just a catch all for key count and bytes count stored in the system-local keyspace for each range.